app/dashboard/static/js/app/view-socs-soc-job-kernel-plan.2020.5.js

```diff
@@ -0,0 +1,433 @@
+                'kernel': result.kernel,
+                'git_branch': result.git_branch,
+                'plan': result.name,
+                'device_type': result.device_type,
+                'lab_name': result.lab_name,
+                'build_environment': result.build_environment,
+                'defconfig_full': result.defconfig_full,
+            });
+
+            idStr = gPanel.createDataIndex(result);
+
+            batchOps.push({
+                method: 'GET',
+                operation_id: idStr,
+                resource: 'test_regression',
+                query: qStr,
+            });
+        }
+
+        batchOps = [];
+        results.forEach(createBatchOp);
+        deferred = request.post(
+            '/_ajax/batch', JSON.stringify({batch: batchOps}));
+
+        $.when(deferred)
+            .fail(error.error, getRegressionsFailed)
+            .done(getRegressionsDone);
+    }
+
+    function getLabResultsFailed() {
+        runsFailed();
+    }
+
+    function getLabResultsDone(response) {
+        updateLabs(response.result);
+    }
+
+    function getLabResults() {
+        var batchOps;
+        var deferred;
+        var params;
+
+        params = {
+            'mach': gSoc,
+            'job': gJob,
+            'kernel': gKernel,
+            'plan': gPlan,
+        };
+
+        function createBatchOp(lab) {
+            var qStr;
+
+            params['lab_name'] = lab;
+            qStr = URI.buildQuery(params);
+
+            batchOps.push({
+                method: 'GET',
+                operation_id: [lab, 'total'],
+                resource: 'count',
+                document: 'test_case',
+                query: qStr,
+            });
+
+            batchOps.push({
+                method: 'GET',
+                operation_id: [lab, 'success'],
+                resource: 'count',
+                document: 'test_case',
+                query: qStr + '&status=PASS',
+            });
+
+            batchOps.push({
+                method: 'GET',
+                operation_id: [lab, 'regressions'],
+                resource: 'count',
+                document: 'test_regression',
+                query: qStr,
+            });
+
+            batchOps.push({
+                method: 'GET',
+                operation_id: [lab, 'unknown'],
+                resource: 'count',
+                document: 'test_case',
+                query: qStr + '&status=FAIL&status=SKIP&regression_id=null',
+            });
+        }
+
+        batchOps = [];
+        Object.keys(gPanel.allLabs).forEach(createBatchOp);
+        deferred = request.post(
+            '/_ajax/batch', JSON.stringify({batch: batchOps}));
+
+        $.when(deferred)
+            .fail(error.error, getLabResultsFailed)
+            .done(getLabResultsDone);
+    }
+
+    function getRunsFailed() {
+        detailsFailed();
+        runsFailed();
+    }
+
+    function getRunsDone(response) {
+        console.log("OK");
```

Comment:
In general looks good, but remove this `console.log` please.

Reply:
Err, sure :innocent:

Reply:
Fixed